cpu/nrf51/include/cpu_conf.h

```diff
@@ -49,6 +49,12 @@ extern "C" {
 #elif defined(CPU_MODEL_NRF51X22XXAB)
 #define FLASHPAGE_NUMOF         (128U)
 #endif
+/* The minimum block size which can be written is 4B. However, the erase
```

Comment:
Since the definitions below are common on both nrf51/nrf52 maybe move them in cpu/nrf5x_common ?

Reply:
In fact, there's no `cpu_conf_common.h` for nrf boards. So let leave it like this.